x/cronos/spec/07_params.md

```diff
@@ -0,0 +1,21 @@
+<!--
+order: 7
+-->
+
+# Parameters
+
+The Cronos module contains the following parameters:
+
+| Key                       | Type   | Default Value                                                |
+| ------------------------- | ------ | ------------------------------------------------------------ |
+| `IbcCroDenom`             | string | `"ibc/6B5A664BF0AF4F71B2F0BAA33141E2F1321242FBD5D19762F541EC971ACB0865"` |
+| `KeyIbcTimeout`           | uint64 | `86400000000000`                                             |
+| `KeyCronosAdmin`          | string | `""`                                                         |
+| `KeyEnableAutoDeployment` | bool   | `false`                                                      |
+
+- `IbcCroDenom` Specifies the IBC token that should be converted to gas token upon arrival automatically.
+
+- `KeyIbcTimeout` The timeout value to use when Cronos module transfer tokens to IBC channels.
+
+- `KeyCronosAdmin` The account that is authorized to manage token mapping through message, empty means no admin, should be a valid bech32 cosmos address if specified.
+- `KeyEnableAutoDeployment` specifies if the auto-deployment feature is enabled.
```

Comment:
what happens when auto-deployment is off and then it's turned on via paramchange proposal? maybe good to add some comments on these parameters whether they are easily "updatable" -- to me, it seems only CronosAdmin can be updated via paramchange... the others look like that they'd need some extra code to be executed for store migrations etc. (so perhaps there should be a warning that they are not easily updatable, but would need some custom upgrade logic?)